content/docs/04-clusters/03-edge/09-site-deployment/06-deploy-cluster.md

```diff
@@ -41,7 +41,7 @@ For learning purposes, you will set up Virtual Machines (VMs) as Edge hosts and
 To complete this tutorial, you will need the following:
 <br/>
 
-* Access to a VMware vCenter environment where you will provision VMs as Edge hosts. You will need the server URL, login credentials, and names of the data center, data store, resource pool, folder, cluster, and DHCP-enabled network.
+* Access to a VMware vCenter environment where you will provision VMs as Edge hosts. You will need the server URL, login credentials, and names of the data center, data store, resource pool, folder, cluster, and DHCP-enabled network. The VMs you will prepare as Edge hosts using the Edge installer ISO image must be DHCP-enabled. 
```

Comment:
This latest addition, how do I, as a user ensure this is true? Can we link to a VMware guide? You may have to break this out into its own bullet.